bin/fetch_ha_env.sh

```diff
@@ -33,14 +33,17 @@ def fetch_env
   staging_env_source = ".env.bt.staging"
   release_env_source = ".env.bt.release"
 
+  resources_commit = mobile_resources_commit()
+  puts "Fetching configuration from pathcheck-mobile-resources commit #{resources_commit}"
+
   dev_env_url =
-  "https://#{token}@raw.githubusercontent.com/Path-Check/pathcheck-mobile-resources/master/environment/#{HA_LABEL}/.env.bt"
+  "https://#{token}@raw.githubusercontent.com/Path-Check/pathcheck-mobile-resources/#{resources_commit}/environment/#{HA_LABEL}/.env.bt"
```

Comment:
you could get rid of that variable if you inline the method, you don't have to add the `()` for a method call, if the definition of the method has no arguments